azure-pipelines.yml

```diff
@@ -4,6 +4,32 @@
 # https://docs.microsoft.com/azure/devops/pipelines/languages/xcode
 jobs:
   - template: ./ci-jobs/functional_test.yml
+  - job: 'unitTests'  # TODO Decide how to handle tox
+    pool:
+       vmImage: 'ubuntu-18.04'
+    strategy:
+      matrix:
+        Python37:
+          python.version: '3.7'
+        Python38:
+          python.version: '3.8'
+        Python39:
+          python.version: '3.9'
+      maxParallel: 2
```

Comment:
why do we limit the amount of parallel tasks?

Reply:
Just copy-pasted one remains, so I can remove.